src/components/RadioTabs.module.css

```diff
@@ -0,0 +1,48 @@
+.radio-tabs {
+  background-color: var(--bs-gray-300);
+  border-radius: 0.25rem;
+  padding: 0.25rem;
+}
+
+:root[data-theme='dark'] .radio-tabs {
+  color: var(--bs-gray-100);
+  background-color: var(--bs-gray-800);
+}
+
+.radio-tab input[type='radio'] {
+  display: none;
+}
+
+.radio-tab label {
+  background-color: white;
+  padding: 0.5rem 1rem;
+  border-radius: 0.25rem;
```

Comment:
This is obviously a nit since it's such a small value here -- but in general, nested border radii should always be smaller than the parent border radii. See [here](https://css-tricks.com/public-service-announcement-careful-with-your-nested-border-radii/) for more details.

Normally a good [rule of thumb](http://web.archive.org/web/20210506121037/https://joshnh.com/weblog/getting-inner-border-radius-just-right/) is that the inner radius should be the outer radius minus the padding between the two elements. This doesn't work since 0.25 (outer radius) minus 0.25 (padding) equals 0. So I'd just go with something like 0.15. It'll look fine as well:

**Before:**
<img width="85" alt="Screenshot 2022-06-06 at 15 59 45" src="https://user-images.githubusercontent.com/10026790/172176236-eb764c09-0f27-4f27-90d3-0f7ed3ef95d0.png">

Look kinda off right?

**After:**
<img width="97" alt="Screenshot 2022-06-06 at 15 59 36" src="https://user-images.githubusercontent.com/10026790/172176268-ffaca68c-f81b-408f-9894-0a346d39ff35.png">

Reply:
love the eye for details :D